src/Concerns/ParallelCommand.php

```diff
@@ -70,7 +72,9 @@ protected function sysctlGetLogicalCoreCount(bool $darwin): int
         // perflevel0 refers to P-cores on M-series, and the entire CPU on Intel Macs
         if ($darwin && $ffi->sysctlbyname('hw.perflevel0.logicalcpu', FFI::addr($cores), FFI::addr($size), null, 0) === 0) {
             return $cores->cdata;
-        } else if ($darwin) {
+        }
+
+        if ($darwin) {
```

Comment:
Think there's no real difference between this version of the code and the previous, but bit like in the previous review, I prefer to have this joined as a single if-elseif block.

Reply:
[`fabc09a` (#1278)](https://github.com/archtechx/tenancy/pull/1278/commits/fabc09a16bf32fca9a42bb04df042a516de5d520)